### PR TITLE
Create authentication

### PR DESCRIPTION
**Pull request by Vacariuc-Alex**

### CODE REVIEW

File: src/components/navbar/AddProduct.tsx [hunk 1]
```diff
@@ -15,10 +17,15 @@ const AddProduct = ((props: AddProductProps) => {
     // Destructure
     const {initialData} = props;
 
+    // Globals
+    const isAuthenticated = !!getItemFromLocalStorageWithExpiration(AUTHENTICATED_USER);
```
Comment (sergheistratu) [inline src/components/navbar/AddProduct.tsx]: what a difference between this variables ?

File: src/components/navbar/AddProduct.tsx [hunk 2]
```diff
@@ -47,23 +54,32 @@ const AddProduct = ((props: AddProductProps) => {
...
-        //if there is initial data, will identify latest id and set as next value, otherwise will be 1 as default
-        if (initialData && initialData.length > 0) {
-            const latestId = parseInt(initialData[initialData.length - 1].id);
-            if (!isNaN(latestId)) {
-                setNextIdAfterLatest(String(latestId + 1));
-            } else {
-                alert(FEATURE_BLOCKED);
-                return;
+        if (isAuthenticated) {
+            if (!initialData) {
```
Comment (sergheistratu) [inline src/components/navbar/AddProduct.tsx]: hard to undertstand can we optimize this part of code ?

Comment (Vacariuc-Alex) [reply src/components/navbar/AddProduct.tsx]: fixed

File: src/components/navbar/AddProduct.tsx (hunk 1 above)
Comment (Vacariuc-Alex) [reply src/components/navbar/AddProduct.tsx]: fixed

**Review by sergheistratu — APPROVED** [review]

**Pull request merged**